spec/system/project_roles_spec.rb

```diff
@@ -29,10 +29,8 @@
     fill_in "data_manager", with: "xxx"
     page.find("body").click
     expect(page.find("button[value=Submit]")).to be_disabled
-    expect(page.find("#data_manager").native.attribute("validationMessage")).to eq "Please select a valid value."
```

Comment:
How is the validation now being tested?

Reply:
Through the button being disabled, but it would be ideal to test the css of the custom validation message